example_test.go

```diff
@@ -237,3 +238,57 @@ func ExampleCredential() {
 		panic(err)
 	}
 }
+
+func ExampleTrace() {
+	// trace.ExecutableTrace is used to define trace hooks that track the
+	// execution of local credential helper executables, and it works with
+	// the credentials.Store interface. The defined hooks will run whenever
+	// a credential helper executable is executed. Multiple ExecutableTrace
+	// can be defined and used, and recently added ExecutableTrace is called
+	// first.
```

Comment:
We might not need these comments at the beginning of the example.

Reply:
FileStore does not involve executables, but it's true that the `ExecutableTrace` tracks on executables and it works on all Stores (that involves executables).

Reply:
> I feel that it's a bit too long to read😂 ... Is it possible make it more concise?
> 
> > trace can be used with all Store implementations
> 
> But it won't work with FileStore.

I feel like the length is fine, [NewStore](https://github.com/oras-project/oras-credentials-go/blob/2afb422fe2259f45846e7bf5b3957634bb46ea58/example_test.go#L83) has a longer comment.

Reply:
Updated this comment. See below for its current display.